src/components/pages/Photos/rover.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import ImageBlock from './ImageBlock';
+import AboutBlock from './AboutBlock';
+import './PhotoPage.css';
+
+export default class Rover extends Component {
+  render() {
+    return (
+      <div className="rover-container">
+        <ImageBlock />
+        <AboutBlock />
+      </div>
+    );
+  }
+}
```

Comment:
This filename should begin with a capital letter as it's a React Component file.